clients/fortnite_api.py

```diff
@@ -35,6 +36,32 @@ async def get_player_stats(ctx, player_name, silent):
     await asyncio.gather(*tasks)
 
 
+async def _get_player_rank(account_info):
+    """Get player rank, latest season? not sure what how fortniteapi handles"""
+    player_name = account_info["readable_name"]
+
+    params = {
+        "username": player_name,
+    }
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(
+            url=RANKED_INFO_LOOKUP_URL,
+            params=params,
+            headers=_get_headers(),
+            raise_for_status=True
+        ) as resp:
+            resp_json = await resp.json()
+
+            if resp_json["result"] is True:
+                if not resp_json["rankedData"] and resp_json["gameId"] != "fortnite":
+                    raise UserStatisticsNotFound(f"Player does not have ranked data: {account_info['readable_name']}")               
+                else:
+                    return [_ranked_data for _ranked_data in resp_json["rankedData"] if _ranked_data["rankingType"] == "ranked-br"][0]
```

Comment:
I think if a player hasn't played ranked BR before, this would fail with IndexError since the list would be empty

We could do like a check before accessing [0], or write a more literal loop like:

```python
for data in ranked_data:
    if data["rankingType"] == "ranked-br":
        return data

return None
```

Reply:
+1 would also recommend writing it out in more lines for readability